src/Symfony/Bridge/Doctrine/Form/Type/EntityType.php

```diff
@@ -46,11 +46,9 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * Return the default loader object.
      *
-     * @param QueryBuilder $queryBuilder
```

Comment:
Shouldn't you keep this phpdoc since the typehint `object` is less restrictive ?